nix/host/scheduled-tasks.nix

```diff
@@ -5,7 +5,10 @@
 { lib, flake, config, ... }:
 let
   networks = [ "preview" "preprod" "mainnet" ];
-  # TODO make this parametric with mode data source
+  # TODO make this parametric with mode data source. NOTE: we have to keep this despite the 
```

Comment:
I'd avoid spawning an issue right know, there are several TODOs (minor things to do) in the code base and I think it's better to batch them in M5